src/Components/Common/components/Menu.tsx

```diff
@@ -0,0 +1,67 @@
+import { Menu, Transition } from "@headlessui/react";
+import { Fragment } from "react";
+
+export default function DropdownMenu(props: {
+  title: string;
+  icon: JSX.Element;
+  children: JSX.Element | JSX.Element[];
+}) {
+  return (
+    <div className="text-right">
+      <Menu as="div" className="relative inline-block text-left w-full">
+        <Menu.Button className="flex w-full lg:justify-between justify-center items-center rounded-[4px] bg-primary-500 px-4 py-2 text-sm font-medium text-white hover:bg-primary-600 focus:outline-none focus-visible:ring-2 focus-visible:ring-white focus-visible:ring-opacity-75">
+          <div className="flex gap-2 items-center">
+            {props.icon}
+            {props.title}
+          </div>
+
+          <i className="ml-2 -mr-1 text-violet-200 hover:text-violet-100 uil uil-angle-down text-2xl font-bold"></i>
+        </Menu.Button>
+        <Transition
+          as={Fragment}
+          enter="transition ease-out duration-100"
+          enterFrom="transform opacity-0 scale-95"
+          enterTo="transform opacity-100 scale-100"
+          leave="transition ease-in duration-75"
+          leaveFrom="transform opacity-100 scale-100"
+          leaveTo="transform opacity-0 scale-95"
+        >
```

Comment:
Replace this with `DropdownTransition` since we need consistent dropdown animation for all types of dropdowns.